src/toil/test/docs/scriptsTest.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import absolute_import
+import unittest
+import os
+import re
+import sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from toil import subprocess
+from toil.test import ToilTest
+from toil.test import needs_cwl
+from toil.test import needs_docker
+from toil.lib.docker import apiDockerCall
+
+class ToilDocumentationTest(ToilTest):
+    """Tests for scripts in the toil tutorials"""
+
+    def tearDown(self):
+        output_files = ["sample_1_output.txt", "sample_2_output.txt", "sample_3_output.txt"]
+        for output in output_files:
+            output_dir = os.path.abspath("scripts/cwlExampleFiles")
+            if os.path.exists(os.path.abspath(os.path.join(output_dir, output))):
+                print("!!!!!!!!!!")
+                os.remove(os.path.abspath(os.path.join(output_dir, output)))
+
+        unittest.TestCase.tearDown(self)
+
+    """Check that the exit code is 0"""
+    def checkExitCode(self, p):
+        out = subprocess.call(["python", p, "file:my-jobstore", "--clean=always"])
```

Comment:
Use `check_call` or `Popen` instead of `call` here.

`call` doesn't wait, so it may return before an exit code actually exists.